sources/TileDB.CSharp/Interop/tiledb_filter_type_t.cs

```diff
@@ -20,5 +20,7 @@ public enum tiledb_filter_type_t
         TILEDB_FILTER_DICTIONARY = 14,
         TILEDB_FILTER_SCALE_FLOAT = 15,
         TILEDB_FILTER_XOR = 16,
+        TILEDB_FILTER_BITSORT = 17,
```

Comment:
I think @ihnorton had said that the bitsort filter is experimental? Should we not expose it in the `FilterType` enum?

Reply:
It was decided to not expose it.